client/app/bundles/Zoon/components/WorkflowCreator.jsx

```diff
@@ -187,6 +188,21 @@ class WorkflowCreator extends React.Component {
     return this.families().length
   }
 
+  showModuleLightbox = (moduleId) => {
+    if (moduleId === null || moduleId === undefined) {
```

Comment:
I wanted to, but crispness of intent took the argument. There's also a slight concern that falseyness might bite us back in the future. Not too worried, since I won't be able to render `0`, or `[]`, etc. The internet is against that in general, but I'm pretty ok with the behavior here.

Mixed feelings here but I have a slight preference for how it is now, since the offense is not too big and might reap rewards. Thoughts?